src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -51,16 +56,16 @@ export default function AccountRecord({
           <TransferDetails account={record.details.account} />
         )}
         {showStatusNote(record, t)}
-        {(record.details.donorCertificate ||
-          record.details.taxDeductibleReceipt ||
-          record.details.giftCertificate) && (
-            <>
-              <div className={styles.title}>{t('downloads')}</div>
-              <div className={styles.detailGrid}>
-                <Certificates recordDetails={record.details} />
-              </div>
-            </>
-          )}
+        {(record?.details?.donorCertificate ||
+          record?.details?.taxDeductibleReceipt ||
+          record?.details?.giftCertificate) && (
+          <>
+            <div className={styles.title}>{t('downloads')}</div>
+            <div className={styles.detailGrid}>
+              <Certificates recordDetails={record.details} />
+            </div>
+          </>
+        )}
```

Comment:
I tested this software with the production backend as I have a conversation donation done there. Unfortunately this of course can not solve the issue if the backend returns a link to a certificate, e.g. https://app.plant-for-the-planet.org/certificate/BYYRKSKQ4891 which then result into an error `TCPDF ERROR: [Image] Unable to get the size of the image: /tmp/certyLJv9q`.